tlatools/org.lamport.tlatools/src/tla2sany/st/SyntaxTreeConstants.java

```diff
@@ -107,12 +107,12 @@ public interface SyntaxTreeConstants {
   int  N_RcdConstructor  =  409 ;
   int  N_RecordComponent  =  410 ;
   int  N_SetEnumerate  =  411 ;
-  int  N_SetExcept  =  412 ;
+//  int  N_SetExcept  =  412 ;        // Removed by AH 05 Oct 2024
```

Comment:
Please use your full name, as future team members may not recognize who "AH" refers to.